RedHatInsights/compliance-frontend
# fix(devtools): User DevAppEntry with redux logger again

This restores a proper dev entry again to enable the redux logging.

**How to test**

1. Verify webpack-dev-server/proxy is still working
2. Visit any page with a SystemsTable (because redux)
3. Verify that redux is logging again

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
